soccer/gameplay/robocup-py.cpp

```diff
@@ -681,6 +704,9 @@ BOOST_PYTHON_MODULE(robocup) {
         .def("nearly_equals", &Geometry2d::Point::nearlyEquals)
         .staticmethod("direction");
 
+    class_<std::vector<Geometry2d::Point>>("vector_Point")
+        .def(vector_indexing_suite<std::vector<Geometry2d::Point>>());
```

Comment:
If this is for the `corners` function, it might be reasonable to just create a wrapper and return and `boost::python::list`. It'll be less "weird" objects to deal with in python.

Reply:
done

Reply:
This part can be removed now I think